constants.gradle

```diff
@@ -15,7 +15,7 @@
  */
 ext {
     groupId = "com.grab.grazel"
-    versionName = "0.4.1-alpha.24"
+    versionName = project.hasProperty("versionName") ? versionName : "0.4.1-alpha.25"
```

Comment:
We should move away from reading property like this and migrate to using Gradle providers later. Requires some effort so can do it later.